lawnchair/src/app/lawnchair/ui/preferences/IconPackPreferences.kt

```diff
@@ -159,6 +165,8 @@ fun IconPackPreferences() {
                     },
                     description = if (themedIconsAvailable.not()) {
                         stringResource(id = R.string.lawnicons_not_installed_description)
+                    } else if(!themedIconPack.contains(iconPackAdapter.state.value) && iconPackAdapter.state.value != "") {
```

Comment:
Move the `if (...)` contents to a variable (like `iconPackSupportsThemedIcon`, `supportsThemedIconPack`, or something else), so you don't have to repeat this code for the above comment:

```kotlin
// somewhere near the top
val supportsThemedIconPack = themedIconPack.contains(iconPackAdapter.state.value) && iconPackAdapter.state.value != ""

// the code in here:
                    } else if(!supportsThemedIconPack) {
```

Reply:
Can't use  both themedIconPack.contains(iconPackAdapter.state.value) && iconPackAdapter.state.value != "" to variable as usuage is different but I have created another variable for better readable as you suggested.